### DISCUSSION TITLE
not giving required files for app 740

### DISCUSSION THREAD

**Discussion by craay123** [SteamRE/DepotDownloader#415] [Q&A]
Attempting to run a slightly older version of CSGO server for some friends of mine who enjoy old counter strike. unfortunately when downloading with the manifest number I want it does not give all the required files to start the server on any platform (windows or linux) 

Steamcmd will give you all the required files and does work but this only gives the newest version. Is this a limitation of depotdownloader or perhaps i'm doing something wrong?

**Comment by wertercatt** [comment]
You need to download matching manifests for both depots [731](https://steamdb.info/depot/731/) and [740](https://steamdb.info/depot/740/)

**Comment by azuisleet** [comment]
Assuming you ran with `-app 740` you would have downloaded the depot 740, which includes srcds.exe and srcds_linux. I don't know the specifics of the CS:GO server, but the two depots 731 and 740 are included in the app 740.

**Comment by wertercatt** [comment]
> Assuming you ran with `-app 740` you would have downloaded the depot 740, which includes srcds.exe and srcds_linux. I don't know the specifics of the CS:GO server, but the two depots 731 and 740 are included in the app 740.

Since they're downloading an older version using a manifest ID, depotdownloader only downloads the specific depot that manifest is for. So they'd need to separately download a matching manifest for the other depot, ideally using -dir to make both depots output to the same folder. If they provided more detail about the manifest they were using or the build they were trying to get, I could provide sample commands to do what they're wanting.
